src.ts/providers/json-rpc-provider.ts

```diff
@@ -60,6 +60,10 @@ export class JsonRpcProvider extends Provider {
         return this.sendJsonRpc('chunk', [chunkId]);
     }
 
+    async gasPrice(blockId: BlockId|null): Promise<GasPrice> {
+        return this.sendJsonRpc('gas_price', [blockId])
+    }
+
     private async sendJsonRpc(method: string, params: any[]): Promise<any> {
```

Comment:
might be good time to make this method public, so that we don't have to expose every RPC

Reply:
Well, given we want to expose all the APIs, it does not seem right to make it public, but I'll let you make the decision. Anyway, it should not be part of this PR.

Reply:
let's wait to see whether others is okay with this

Reply:
@frol there is really no practical benefit to not make it public IMO. There can be various reasons why somebody has to stay on older version of library, but still call newest RPC. This seems as practicality vs theoretical correctness trade-off.

Reply:
@vgrichina Double negative is hard to reason about in English. I am neutral about promoting it to the public interface, but I want to make it separately from this PR :man_shrugging:

Reply:
>  but I want to make it separately from this PR 🤷‍♂

seems reasonable

Reply:
Just for the reference: https://github.com/near/near-api-js/pull/272